clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -1303,6 +1311,10 @@ boolean isRetry() {
             return isRetry;
         }
 
+        int Attempts() {
```

Comment:
nit: Change `Attempts()` to `attempts()`

Reply:
Good catch. Refactored.